oshc/main/templates/session.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html' %} {% load static %} {% block content %}
+
+<iframe id="typeform-full" width="100%" height="100%" frameborder="0" src="https://vaibhavsingh.typeform.com/to/Y4ioHZ"></iframe>
```

Comment:
@tapasweni-pathak Please provide me the necessary credentials so that i can change this link.